conversation_reconstruction/dataflow/write_page_states_to_spanner/write_utils/write.py

```diff
@@ -42,6 +46,48 @@ def insert_data(self, table_id, data):
       with self.database.batch() as batch:
         batch.insert(
             table = table_id,
-            columns = self.table_columns[table_id],
-            values = data)
+            columns = sorted(self.table_columns[table_id].keys()),
+            values = self.convert_data(table_id, data))
       return 'Inserted data.'
+
+    def _convert(self, datatype, data):
```

Comment:
This function does not use self. so I don't know if it belong in this class. If it does, you want it to be @static. 

I think this also should be named more specifically: it converts a bigquery value. So a better name would be: 
`convert_value_bq_to_spanner`